src/executor.rs

```diff
@@ -177,7 +177,6 @@ fn invoke_dynamic(
                     .to_bytes(&mut invoke_data)?;
             }
             CoreTypeConcrete::BuiltinCosts(_) => {
-                // This builtin should never be an argument but just in case.
```

Comment:
Can it be an argument now?

Reply:
yes, according to some tests with replay